app/graphql/types/query_type.rb

```diff
@@ -24,6 +24,8 @@ class QueryType < Types::BaseObject
     field :customer_usage, resolver: Resolvers::Customers::UsageResolver
     field :customer_invoices, resolver: Resolvers::Customers::InvoicesResolver
     field :customer_portal_invoices, resolver: Resolvers::CustomerPortal::InvoicesResolver
+    field :customer_portal_user, resolver: Resolvers::CustomerPortal::CustomerResolver
+    field :customer_portal_organization, resolver: Resolvers::CustomerPortal::OrganizationResolver
```

Comment:
GraphQL/OrderedFields: Fields should be sorted in an alphabetical order within their section. Field `customer_portal_organization` should appear before `customer_portal_user`.